api/types/types.go

```diff
@@ -185,6 +185,9 @@ type Version struct {
 type Info struct {
 	ID                 string
 	Containers         int
+	ContainersRunning  int
+	ContainersPaused   int
+	ContainersStopped  int
```

Comment:
Should we use a map, to include number of containers per state-x? May be more future-proof (and allows, e.g. number of "dead" containers)

@duglin wdyt?